### ISSUE TITLE
Manual does not have a clear statement how let binding operator are desugared

### ISSUE THREAD

**Issue by nojb** [ocaml/ocaml#9430]
Unless I am mistaken, the manual has no clear specification of how let binding operators are desugared (although it does contain examples).

I would have expected to find somewhere formulas along the lines:
```
letOP v = e1 in e2 => (letOP) e1 (fun v -> e2)

letOP v1 = e1 andOP v2 = e2 in e3 => (letOP) ((andOP) e1 e2) (fun (v1, v2) -> e3)
```

**Comment by Octachron** [comment]
A potential difficulty is that binding operators are not just a syntactic sugar since they appear in the ASTs until the translation to Lambda.

**Comment by nojb** [comment]
But from a user perspective, is there any difference?

**Comment by lpw25** [comment]
I don't really see the difference between what you have written and the examples already in the manual. Yours just looks like an example with very generic variable names to me.

**Comment by gasche** [comment]
Well so maybe we can include the non-generic example and also the generic example and make you both happy :-)

**Comment by damiendoligez** [comment]
> Yours just looks like an example with very generic variable names to me.

Then we're missing a statement that the examples are fully general? Something to tell the user that this is the whole spec and there's nothing more to it. A mere example doesn't fully convey that meaning.

**Comment by nojb** [comment]
This will be fixed by #11291
